src/analysis/normed_space/basic.lean

```diff
@@ -1144,6 +1145,9 @@ instance prod.semi_normed_ring [semi_normed_ring β] : semi_normed_ring (α × 
         ... = (∥x∥*∥y∥) : rfl,
   ..prod.semi_normed_group }
 
+instance {n m : Type*} [fintype n] [fintype m] : semi_normed_group (matrix n m α) :=
+pi.semi_normed_group
```

Comment:
This could do with a comment like the one on `pi.semi_normed_group`, as well as a lemma that says:
```
∥A∥ ≤ r ↔ ∀ i j, ∥A i j∥ ≤ r
```